### DISCUSSION TITLE
More colour options on Button Toggle Group

### DISCUSSION THREAD

**Discussion by angela-gibson** [jpmorganchase/salt-ds#2951] [Feedback]
When I've been presenting my designs on Figma using SALT components, there is often feedback on the Button Toggle Group not being clear what's selected if it's between 2 options.  Because you can't choose the highlight colour, the default is grey.  Users don't know if the grey indicates that it's not selected or it is selected.  This keeps coming up in feedback.  Would we be able to have another option to choose from, like in UITK there was the blue option?

**Comment by mark-tate** [comment]
converted to an issue
https://github.com/jpmorganchase/salt-ds/issues/3361
